scripts/install.sh

```diff
@@ -25,3 +25,11 @@ if [[ "${TOX_ENV}" == "pypy" ]]; then
     pyenv install pypy-2.6.0
     pyenv global pypy-2.6.0
 fi
+
+if [[ "${TOX_ENV}" == "gae" && ! -d $HOME/google_appengine ]]; then
+    cd $HOME
+    wget https://storage.googleapis.com/appengine-sdks/featured/google_appengine_1.9.25.zip
```

Comment:
We don't want the version to be hard-coded. Luckily @craigcitro wrote us a nice script: `scripts/fetch_gae_sdk.py` which takes an optional argument to specify where to download it to.